github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```diff
@@ -186,6 +186,10 @@ tide:
         dashboard:
           skip-unknown-contexts: true
           from-branch-protection: true
+     nmstate:
+      repos:
```

Comment:
It has bad identation, it's missing an space.

Reply:
thanks for the heads up! it is not required for the context_options to be applied but good to have it fixed, already pushed the changes